vendor/milesight-iot/em500-smt-codec.yaml

```diff
@@ -0,0 +1,23 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: em500-smt.js
+    - description: Soil Moisture Sensor (example 1) - Milesight IoT
+      input:
+        fPort: 85
+        bytes: [0x01, 0x75, 0x5C, 0x04, 0x68, 0x65]
```

Comment:
are you sure about bytes? data only reads battery and humidity, please check with .js file em500-smt.js